tracee/tracee.go

```diff
@@ -473,22 +473,33 @@ func (t *Tracee) processEvents() {
 				t.handleError(err)
 				continue
 			}
+			argsNames := make([]string, ctx.Argnum)
+			argsTags := make([]uint8, ctx.Argnum)
 			args := make([]interface{}, ctx.Argnum)
 			for i := 0; i < int(ctx.Argnum); i++ {
-				args[i], err = readArgFromBuff(dataBuff)
+				argTag, argVal, err := readArgFromBuff(dataBuff)
 				if err != nil {
 					t.handleError(err)
 					continue
 				}
+				argsTags[i] = argTag
+				args[i] = argVal
+				argName, ok := argNames[argTag]
+				if ok {
+				       argsNames[i] = argName
+				} else {
+				       t.handleError(err)
+				       continue
```

Comment:
do we want to fail here or to just skip the tag parsing? (I'm not sure myself)

Reply:
This is definetely an error. If an argtag doesn't have an argName, this means something went wrong in the event decoding, as this shouldn't happen